.github/workflows/push.yml

```diff
@@ -0,0 +1,25 @@
+name: Push
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set Up Linux
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libv4l-dev libudev-dev libasound2-dev
```

Comment:
Since we're not using any of the functionality that relies on these libraries, I think I can actually get rid of these requirements fairly easily, but that's for another PR.